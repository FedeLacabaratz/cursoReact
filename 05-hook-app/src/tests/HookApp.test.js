import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import HookApp from '../HookApp';

describe('pruebas en <HookApp />', () => {
    
    let wrapper = shallow(<HookApp />);

    beforeAll(() => {
        jest.clearAllMocks();
        wrapper = shallow(<HookApp />);
    });

    test('It should print the component correctly', () => {
        
        expect(wrapper).toMatchSnapshot();
    });

});
