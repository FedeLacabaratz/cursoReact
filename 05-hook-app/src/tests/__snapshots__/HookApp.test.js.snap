// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <HookApp /> It should print the component correctly 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
